### DISCUSSION TITLE
My Flowrate is not working when building from 1.3.5b

### DISCUSSION THREAD

**Discussion by doubleO00** [Jyers/Marlin#1451] [Q&A] [answered]
Hey guys

I needed to make a build for a larger bed conversion but I cant seem to get the flowrate to work, it shows up at 100% but no matter what I dial it down to there is no change to the printed cube.

Using 1.3.5b as a base and jyers  ev2 config bltouch 3x3

**Comment by Jyers** [comment, marked as answer]
Could perhaps be an issue with the old marlin source. Try the latest source and see if it helps. A new release should come out soon.
